modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -25,6 +25,9 @@ The `domain` value must be unique among all Ingress controllers and cannot be up
 
 If empty, the default value is `ingress.config.openshift.io/cluster` `.spec.domain`.
 
+|`appsDomain`
```

Comment:
This got documented in the wrong place: https://bugzilla.redhat.com/show_bug.cgi?id=1967592